chemprop/nn/message_passing/base.py

```diff
@@ -182,13 +182,11 @@ def forward(self, bmg: BatchMolGraph, V_d: Tensor | None = None) -> Tensor:
             an optional tensor of shape ``V x d_vd`` containing additional descriptors for each atom
             in the batch. These will be concatenated to the learned atomic descriptors and
             transformed before the readout phase.
-            **NOTE**: recall that ``V`` is equal to ``num_atoms + 1``, so ``V_d`` must be 0-padded
-            in the 0th row.
```

Comment:
So are we not doing 0th row padding anymore?

Reply:
Yes David told me the zeroth row padding is "a remnant of the old scatter/gather ops in pure torch. With the backend move to `torch_scatter` it's no longer necessary". 
I noticed just now message_passing proto also has this note, so I will add that change.